extensions/vertx-core/runtime/src/main/java/io/quarkus/vertx/core/runtime/config/AddressResolverConfiguration.java

```diff
@@ -0,0 +1,31 @@
+package io.quarkus.vertx.core.runtime.config;
+
+import java.util.OptionalInt;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class AddressResolverConfiguration {
+
+    /**
+     * The maximum amount of time in seconds that a successfully resolved address will be cached.
+     * <p>
+     * If not set explicitly, resolved addresses may be cached forever.
+     */
+    @ConfigItem(defaultValue = "2147483647")
```

Comment:
I don't think you need to specify a default value here for any of these attributes, since you're already handling that in the recorder class

Reply:
Hi @gastaldi this is purely for the generated doc as suggested in my feedback here https://github.com/quarkusio/quarkus/pull/13238#discussion_r523438858

Reply:
@machi1990 ah sorry, I missed that. 

In this case you don't need the types to be optional, just use `int` directly

Reply:
+1, missed it also.

Reply:
OK, I have replaced the OptionalInt with primitive int values.